common/templates/common/docs/catalog.html

```diff
@@ -10,13 +10,92 @@ <h2 class="red-text text-darken-4">Catalog</h2>
 <h4 class="red-text text-darken-4">Course Lookup</h4>
 
 <h5>/courses/all <span class="grey-text">(GET)</span></h5>
-<p>Returns a JSON list of all courses in the current version of the catalog, in numerically sorted order. Takes Boolean query parameter <span class="code">full</span>, indicating whether to return the full set of information for each subject or an abbreviated version.</p>
+<p>Returns a JSON list of all courses in the current version of the catalog, sorted by subject ID as strings. Takes Boolean query parameter <span class="code">full</span>, indicating whether to return the full set of information for each subject or an abbreviated version.</p>
+
+<p>Each entry in the abbreviated version is a dictionary that has the following keys:</p>
+
+<ul class="collection">
+  <li class="collection-item"><span class="code">subject_id</span> - subject ID string, e.g. "6.1200".</li>
+  <li class="collection-item"><span class="code">title</span> - the full title, e.g. "Mathematics for Computer Science".</li>
+  <li class="collection-item"><span class="code">total_units</span> - number of units.</li>
+  <li class="collection-item"><span class="code">offered_fall</span>, <span class="code">offered_IAP</span>, <span class="code">offered_spring</span>, <span class="code">offered_summer</span> - whether the class is offered in the given semester.</li>
+  <li class="collection-item"><span class="code">public</span> - whether the subject is a custom subject. For catalog subjects, this is always false.</li>
```

Comment:
Isn't `public` always true for catalog subjects (and false for custom subjects)?

Reply:
oh so true bestie